src/lib/eet/eet_api.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _EFL_EET_API_H
+#define _EFL_EET_API_H
+
+#ifdef EET_API
+#error EET_API should not be already defined
+#endif
+
+#ifdef _WIN32
+# ifndef EFL_STATIC
+#  ifdef EET_BUILD
+#   define EET_API __declspec(dllexport)
+#  else
+#   define EET_API __declspec(dllimport)
+#  endif
+# else
+#  define EET_API
+# endif
+# define EET_API_WEAK
```

Comment:
`EET_WEAK_API` is not used. Should it be defined?

Reply:
For now, the pattern I'm following is the one established by @felipealmeida, where it was defined.
But I agree with you on that, don't think we need to define it with it isn't used.

Reply:
I would like to listen to other opinions here, guys.

Reply:
I think we should apply this patch as is;
if we are going to tailor each API to contain only the used symbols I think it would be more interest to do it when all this API idea gets more mature.

Reply:
are you okay with that @JPTIZ ?

Reply:
I'm ok, yes.